src/components/pages/deploy/viewer.tsx

```diff
@@ -43,36 +40,22 @@ const DeployWidget = styled.div`
   align-items: stretch;
 `;
 
-const StyledBackdrop = withStyles(() => ({ root: { zIndex: 999 } }))(Backdrop);
-
-export interface Props {
-  onSubmit?: (event: React.FormEvent<HTMLFormElement>) => Promise<void>;
-}
-
-/**
- * All component props are passed down into a local context
- * Every fragment takes what is needs from it
- */
-function component(props: Props): JSX.Element {
-  const isSubmitting = useRecoilValue(isSubmittingAtom);
-
+function component(): JSX.Element {
```

Comment:
Can you rename this function to something more meaningful?

Reply:
You mean the `component` function?

That's what I've been using as a standard for all components before applying the `memo` to it and exporting with the proper component name. (See https://github.com/gnosis/safe-cmm-app-react/blob/development/src/components/pages/deploy/viewer.tsx#L78)

I was using before the underscore (for example, `_DeployPageViewer`) version which -- I agree with you -- was not very nice, and had since been using `component`.

I like the name `component` because makes clear what is the main component in the module.
An alternative to name the component `component` is to wrap an anonymous function in `memo` directly and give it the final name.

Have any better suggestions?
In any case, I'd rather do this change in another PR, as I have all the memo'd components following this pattern.